nix/haskell-overlay.nix

```diff
@@ -1,3 +1,2 @@
 self: super: {
-  hashable = super.hashable_1_4_0_2;
```

Comment:
Nice that the overlay was resolved with the nix pin update!

Reply:
I was quite surprised tbh.
Building hashable took forever, so I tried taking it out, and it worked.